Makefile

```diff
@@ -3,6 +3,8 @@ IMAGE_LAMBDA_GO=eawsy/aws-lambda-go-shim
 GOPATH ?= $(HOME)/go
 HANDLER ?= handler
 PACKAGE ?= package
+AWS_ACCOUNT_NUMBER ?= $(shell aws iam list-users --output text | head -1|  cut -f2 | cut -f5 -d:)
```

Comment:
@thomasv314 the AWS CLI was added to for debugging purposes because I couldn't get serverless to work in the beginning. But now that @cristim has fixed the actual issue, this feature can be removed.